linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -68,6 +71,9 @@ class HttpInitializer extends ProtocolInitializer.Simple {
       .replace(Headers.Ctx.serverModule.role, Headers.Ctx.serverModule)
       .prepend(http.ErrorResponder.module)
       .prepend(http.StatusCodeStatsFilter.module)
+      // ensure the server-stack framing filter is placed below the stats filter
+      // so that any malframed requests it fails are counted as errors
+      .insertAfter(StatusCodeStatsFilter.role, FramingFilter.serverModule)
```

Comment:
Okay, looks like `StatusCodeStatsFilter` is actually at the top of the stack, and `RequestStats` towards the bottom. Fixed in 33d05c3c3b255f44fb42b05ff119f88afc51bc19